vhd/packer/configure-windows-vhd.ps1

```diff
@@ -49,8 +49,13 @@ function Get-ContainerImages {
                 "mcr.microsoft.com/windows/nanoserver:1809",
                 "mcr.microsoft.com/oss/kubernetes/pause:1.4.1",
                 "mcr.microsoft.com/oss/kubernetes/pause:3.4.1",
-                "mcr.microsoft.com/oss/kubernetes-csi/livenessprobe:v2.0.1-alpha.1-windows-1809-amd64",
-                "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v1.2.1-alpha.1-windows-1809-amd64")
+                "mcr.microsoft.com/oss/kubernetes/azure-cloud-controller-manager:v1.0.8",
+                "mcr.microsoft.com/oss/kubernetes/azure-cloud-controller-manager:v1.1.4",
+                "mcr.microsoft.com/oss/kubernetes-csi/azuredisk-csi:v1.10.0",
+                "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v2.4.0",
+                "mcr.microsoft.com/oss/kubernetes-csi/livenessprobe:v2.5.0",
+                "mcr.microsoft.com/oss/azure/secrets-store/provider-azure:0.0.12",

```

Comment:
I am not sure if this secrets-store provider image is needed on the Windows VHD for agent node. Please advise @jackfrancis

Reply:
Are you confident that these container images are multi-arch? (linux and windows)

Reply:
I verified the cloud-node-manager and csi images are working fine on Windows nodes. The secrets-store images are not needed on Windows as the secrets-store pods are only on linux nodes.